### PR TITLE
L rh0zw du/add address

### PR DESCRIPTION
**Pull request by SergiiGoncharov13**

### CODE REVIEW

File: add_address.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
+def add_address(data: str):
+    name, address = split_data(data)
+    if book.name:
```
Comment (ArturFartukh) [inline add_address.py]: if name not in book.data.keys()

File: add_address.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        raise ValueError('This user not in contact book')
+    if book.address:
+        raise ValueError('This contact already exist address')
+    record = add_address(address)
+    book.add_record(record)
```
Comment (ArturFartukh) [inline add_address.py]: `record = book[name]`
book.add_record(record) - видалити

Comment (SergiiGoncharov13) [reply add_address.py]: fix

File: add_address.py (hunk 1 above)
Comment (SergiiGoncharov13) [reply add_address.py]: fix

**Review by ArturFartukh — APPROVED** [review]

**Pull request merged**